examples/windows-node-groups/README.md

```diff
@@ -24,7 +24,7 @@ git clone https://github.com/aws-samples/aws-eks-accelerator-for-terraform.git
 to initialize a working directory with configuration files
 
 ```bash
-cd examples/eks-cluster-with-windows-support
```

Comment:
We may need to replace path on other places besides this file, see https://github.com/aws-samples/aws-eks-accelerator-for-terraform/blob/main/docs/node-groups.md?plain=1#L191

Reply:
updated